rails-commenting.rb

```diff
@@ -39,15 +39,15 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # 6)
+      # 6) If the requirements aren't met to delete a post based off of its id then this will redirect the user back to where they started
       redirect_to blog_post_path(@post)
     end
   end
 
-  # 7)
+  # 7) Private is the keyword that protects all methods below. The strong params are always listed below it.
```

Comment:
Totally. Maybe add in what private does, and why we would want to protect the methods.